integration/src/main/java/scripts/testscripts/ListWorkspaces.java

```diff
@@ -0,0 +1,93 @@
+package scripts.testscripts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.model.WorkspaceDescription;
+import bio.terra.workspace.model.WorkspaceDescriptionList;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import scripts.utils.ClientTestUtils;
+import scripts.utils.WorkspaceAllocateTestScriptBase;
+
+/**
+ * This test exercises WSM's listWorkspaces endpoint. Note that because this test reads the full
+ * set of a user's workspaces, it is not thread safe and only one instance of this test should
+ * run against a server at once.
+ */
+public class ListWorkspaces extends WorkspaceAllocateTestScriptBase {
+
+  private UUID workspaceId2;
+  private UUID workspaceId3;
+  private WorkspaceApi secondUserApi;
+
+  @Override
+  protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi workspaceApi) throws Exception {
```

Comment:
it helps me to rename `workspaceApi` to `user1WorkspaceApi` or simlar.

Reply:
done